### PR TITLE
use possibleTypes in identifierTagger of JS

### PR DESCRIPTION
**Pull request by khemrajrathore**
We need to do a deep analysis into when to use `possibleTypes` and `typeFullName`, `methodFullName`, `dynamicTypeHintFullName` etc

We may also need to make changes in audit report or any other place if we see a gap

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ ThisBuild / version      := sys.env.getOrElse("BUILD_VERSION", "dev-SNAPSHOT")
 // parsed by project/Versions.scala, updated by updateDependencies.sh
 
 val cpgVersion        = "1.6.10"
-val joernVersion      = "2.0.323"
+val joernVersion      = "2.0.324"
```
Comment (pandurangpatil) [inline build.sbt:11]: Once the comparison report looks positive try with the latest version `2.0.327`

Comment (pandurangpatil) [reply build.sbt:11]: @khemrajrathore Were you able to resolve the error?

Comment (pandurangpatil) [reply build.sbt:11]: Looks like the latest version of overflowdb is `1.192` did you try with `1.92` or `1.192`?

Comment (khemrajrathore) [reply build.sbt:11]: Yes, tried with `192` misspelt while copying

**Pull request merged**